ghost/admin/app/components/posts/analytics.js

```diff
@@ -302,6 +355,35 @@ export default class Analytics extends Component {
         this.mentions = yield this.store.query('mention', {limit: 5, order: 'created_at desc', filter});
     }
 
+    @task
+    *fetchPostCountTask() {
+        if (!this.post.emailOnly) {
+            const result = yield this.store.query('post', {filter: 'status:published', limit: 1});
+            let count = result.meta.pagination.total;
+
+            count += 1; // account for the new post
```

Comment:
I don't think we need to account for the new post here as the store will be updated as soon as the post is published. Please test this.

Reply:
I tested it with a new account. We don't need to do +1 here. It says 5 posts when I have 4 posts published. 
<img width="644" alt="image" src="https://github.com/user-attachments/assets/3881abfc-79d3-4724-bb36-fdc5f9fd6bad">
<img width="1329" alt="image" src="https://github.com/user-attachments/assets/47b3136a-46d4-44c8-b0f1-bdc3dd81de49">